### PR TITLE
Completed prep work for netty tests 

### PR DESCRIPTION
**Pull request by vanichitkara**
Completed the prep work for netty tests:
Added README.md
Added test.properties

Please review my PR for netty test prep work and suggest changes and improvements.

### CODE REVIEW

File: external/netty/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# External Netty Tests
+
+Netty tests are part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests.
```
Comment (sophia-guo) [inline external/netty/README.md:3]: As AdoptOpenJDK has moved to Eclipse as Adoptium please update references to AdoptOpenJDK to be Adoptium. Thanks!

Comment (vanichitkara) [reply external/netty/README.md]: I have changed all the references of AdoptOpenJDK to Adoptium. If there is still any reference that I missed my eye, please let me know.

File: external/netty/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# External Netty Tests
+
+Netty tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests.
```
Comment (smlambert) [inline external/netty/README.md:3]: One more request, please add an extra sentence that says:
Netty test material is pulled from the [netty](https://github.com/netty/netty.git) repository.

*please also hyperlink netty to the value from github_url in the test.properties file

Comment (vanichitkara) [reply external/netty/README.md]: Done the following change, please review it.

**Review by sophia-guo — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]

**Pull request merged**